<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceInfoDialog</class>
 <widget class="QDialog" name="ServiceInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_charging_event_timestamp"/>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBox_ServiceFinishResult"/>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="checkBox_realtime_charged_energy">
       <property name="text">
        <string>realtime_charged_energy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_service_id">
       <property name="text">
        <string>service_id</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_service_id"/>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_ServiceEvent">
       <property name="text">
        <string>ServiceEvent</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="checkBox_charged_energy_total">
       <property name="text">
        <string>charged_energy_total</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_realtime_charged_energy"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBox_charging_event_timestamp">
       <property name="text">
        <string>charging_event_timestamp</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_charged_energy_total"/>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBox_charging_connector_id">
       <property name="text">
        <string>charging_connector_id</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBox_ServiceFinishResult">
       <property name="text">
        <string>ServiceFinishResult</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_ServiceEvent"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_charging_connector_id"/>
     </item>
     <item row="7" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_save">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton_quit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
